apps/daffio/src/app/docs/design/components/component-content/component-content.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  input,
+} from '@angular/core';
+
+import { DAFF_ARTICLE_COMPONENTS } from '@daffodil/design/article';
+import { DAFF_TABS_COMPONENTS } from '@daffodil/design/tabs';
+import {
+  DaffDocKind,
+  DaffPackageGuideDoc,
+} from '@daffodil/docs-utils';
+
+import { DaffioSafeHtmlPipe } from '../../../../core/html-sanitizer/safe.pipe';
+import { DaffioDocArticleModule } from '../../../components/doc-article/module';
+import { DaffioDocsDynamicContent } from '../../../dynamically-renderable-content/type';
+
+@Component({
+  selector: 'daffio-docs-design-component-content',
+  templateUrl: './component-content.component.html',
```

Comment:
missing template file

Reply:
yeah sorry this shouldn't have been included in this PR, it has been removed